autoload/wintabs/ui.vim

```diff
@@ -185,11 +185,14 @@ function! s:get_tab_name(n)
   if empty(title)
     let buflist = tabpagebuflist(a:n)
     let winnr = tabpagewinnr(a:n)
-    let title = bufname(buflist[winnr - 1])
-    if empty(title)
+    let buffer = bufname(buflist[winnr - 1])
```

Comment:
buffer in wintabs has always meant bufnr, if you want to use a better name, use [file]name or [file]path.